saleor/graphql/account/mutations/base.py

```diff
@@ -129,7 +129,7 @@ def perform_mutation(cls, _root, info, **data):
             )
 
         try:
-            user = models.User.objects.get(email=email)
+            user = models.User.objects.get(email=email, is_active=True)
```

Comment:
What concerns me here, is that we say the account doesn't exist to the client, but the account is actually disabled

Reply:
Yes, we discussed it with the team and it would be better to raise a different error. We should leave this piece as it was before and after the user is fetched we should add code that checks if the account is inactive and raises an error if it is. I think we need to add a new error code for that: `AccountErrorCode.INACTIVE`.